src/workers/orbit/importActivities.js

```diff
@@ -0,0 +1,66 @@
+import { Worker, Queue } from "bullmq";
```

Comment:
@steeve-orbit This is the single-file pattern. One advantage is that just this worker can be started from a command line script or by importing the file, without causing other workers to start up.

Reply:
> without causing other workers to start up

🤔 I don't think there's a huge benefit in doing that since, as far as I know, it's quite cheap to have the workers sitting idle. Also we may want to start them ASAP anyway, we could still have unprocessed jobs on the queues from the previous deployment.